### PR TITLE
refactor arrowhead code

### PR DESCRIPTION
**Pull request by katieschilling**

### CODE REVIEW

File: arrowhead/arrowhead.rb [hunk 1]
```diff
@@ -17,17 +17,20 @@ class Arrowhead
...
-      raise "Unknown region, please provide a valid region."
-    end
+    shapes = CLASSIFICATIONS[region]
+    raise "Unknown region, please provide a valid region." unless shapes
+    identify_shape(shapes, shape)
+  end
+
+  def self.identify_shape(shapes, type)
+    shape = shapes[type]
+    shape_error unless shape
```
Comment (incazteca) [inline arrowhead/arrowhead.rb]: Is there a specific reason you moved the raise into a separate method? Keep the error message close to the actual error.

Comment (katieschilling) [inline arrowhead/arrowhead.rb]: Line too long error. Moved it at polish dan's suggestion.

**Comment by incazteca** [comment]
@katieschilling Looks good I would probably remove the shape_error method as the error being thrown is not the one you created but a different error ```arrowhead.rb:27:in `identify_shape': undefined local variable or method `shape_error' for Arrowhead:Class (NameError)```

File: arrowhead/arrowhead.rb (hunk 1 above)
Comment (djkotowski) [inline arrowhead/arrowhead.rb]: I don't think this is too bad given `shape_error` is near `identify_shape`.  Another option would be to declare a constant with the long error message.

File: arrowhead/arrowhead.rb [hunk 2]
```diff
@@ -17,17 +17,20 @@ class Arrowhead
...
+    "You have a(n) '#{shape}' arrowhead. Probably priceless."
+  end
+
+  def shape_error
```
Comment (djkotowski) [inline arrowhead/arrowhead.rb]: Make this a class method.

**Comment by djkotowski** [comment]
This looks good.  Take a look at the comments, but no need for a resubmission.  Nice work!

**Pull request closed without merging**